src/components/find-on-shelf/FindOnShelfModal.tsx

```diff
@@ -78,15 +78,38 @@ const FindOnShelfModal: FC<FindOnShelfModalProps> = ({
   });
 
   // Sorting of the data below to show branches & manifestations in the correct order.
-  const finalDataAlphabetical = finalData.sort(
-    (a: ManifestationHoldings, b: ManifestationHoldings) => {
-      return a[0].holding.branch.title.localeCompare(
-        b[0].holding.branch.title,
-        "da-DK"
-      );
+  function orderManifestationHoldingsAlphabetically(
+    a: ManifestationHoldings,
+    b: ManifestationHoldings
+  ) {
+    return a[0].holding.branch.title.localeCompare(
+      b[0].holding.branch.title,
+      "da-DK"
+    );
+  }
+  const availableManifestationHoldings = finalData.filter(
+    (manifestationHolding: ManifestationHoldings) => {
+      return isAnyManifestationAvailableOnBranch(manifestationHolding);
+    }
+  );
+  const unavailableManifestationHoldings = finalData.filter(
+    (manifestationHolding: ManifestationHoldings) => {
+      return !isAnyManifestationAvailableOnBranch(manifestationHolding);
     }
   );
```

Comment:
Instead of filtering the same array twice we could use something like `_.partition()` to split the array in into available and unavailable parts.

The performance gained from avoiding this is negligeable so the primary point would be to shorten the code. @spaceo has opinions on when/if we should use Lodash but [`_partition()` is at least not one of the functions that are easily replaced](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore).

Reply:
@sinejespersen, @spaceo will talk to you about this :)

Reply:
@Adamik10 I've talked to Mikkel, Lodash partition is fine by me